### ISSUE TITLE
Angular example broken in 4.0.x

### ISSUE THREAD

**Issue by PhiLhoSoft** [Reactive-Extensions/RxJS#963]
The interesting article https://medium.com/@gerard.sans/angular-introduction-to-reactive-extensions-rxjs-a86a7430a61f shows how to use RxJS with AngularJS. It provides a plunk which works fine, using RxJS 2.5.3: http://plnkr.co/edit/P8dELQZ6HlglomXSvOcj
I tried it with RxJS 4.0.0 (and 0.1 and 0.2) and it doesn't behave correctly.
It works fine with RxJS 3.1.1.

Symptoms:
If I type `jump` quickly (less than 1 s), only the search for J is shown.
If I type more slowly (more than 1 s between keys), it shows the search results from the latest string, except if I type two letters too quickly: the latest isn't searched.

I fear I am too new to RxJS to help furthermore. At least, the bug from `fromEvent` taught me that 4.0.x has some regression bugs... :smile:

**Comment by paulpdaniels** [comment]
The methods `throttle` and `debounce` were switched to be more consistent with their actual meaning. So if you change `throttle` to `debounce` it should work with 4.0.2.

**Comment by PhiLhoSoft** [comment]
Ah, I missed that when I have read the [changelog](https://github.com/Reactive-Extensions/RxJS/releases) before making the issue.
Actually, I checked again, it is not mentioned there.
Where do these breaking changes are listed?

Anyway, [I just tested with debounce and it worked again with 4.0.2](http://plnkr.co/edit/ounvgRGwWBbs7O7aR5wN), so I will close this issue. Will rename it so it can be spotted by people having a similar problem.
